### DISCUSSION TITLE
New release : lots of false positives

### DISCUSSION THREAD

**Discussion by HAOccassionalUser** [blakeblackshear/frigate#1988] [Ask A Question]
Hi
I have upgraded to the  latest release (running Frigate on HA)  and have  seen the following:
1. CPU usage on an average has gone up from around 22% to 36%
2. Lots more peak CPU alerts (alerts triggered when CPU usage is more than 90% for a minute of more)
2. Lots of triggers for objects that do not move. e.g. neighbours parked car now triggers an alert, whereas before this release, the car would not have triggered an alert. The consequence of this is that automations associated with these events , like phone alerts, outside lights etc, trigger every 5 to 10 mins.. 

Is there a way to go back to the previous release?
regards
Tom

**Comment by blakeblackshear** [comment]
If you are using a Coral, did you add it to your config? The latest version defaults to a CPU detector. The model is identical to the last version, so there is no reason false positives would go up.

**Reply by HAOccassionalUser** [reply]
Here is the coral config. 
```yaml
detectors:
  coral:
    type: edgetpu
    device: usb
ffmpeg:
  global_args: -hide_banner -loglevel warning
  hwaccel_args:
    - -c:v
    - h264_v4l2m2m
```

I havent made any changes, other than to comment out the height and width which were causing errors (I understand you moved that to under detect) and comment out a line for the ffmeg config as follows
Commented out    clips: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy -an

I was getting a lot of errors about clips and commenting out this line has stopped those errors. 

For information, I dont use clips, just the snapshots

**Reply by blakeblackshear** [reply]
The only thing I can think of is the motion detection settings are now dynamically calculated based on your camera's resolution. If the new defaults are more sensitive, that could be causing the issues you describe. You say "false positive", but it sounds like you mean more detections of stationary objects.

**Reply by HAOccassionalUser** [reply]
yes... I guess you are right in that it is more detection of stationary objects.  Its picking up all cars in the camera view and triggering identification day and night. Whereas, with the previous version, it was only triggering on motion of defined objects. And that was absolutely superb in that it worked so seamlessly, correctly and out of the box. I am having to disable my automation now..
Are there are any settings I can use to reduce this? You mentioned dynamic calculation, anything there that I can influence..

**Reply by blakeblackshear** [reply]
Can you post the JSON config from the debug page? You should be able to configure them to be the same as before.

**Reply by HAOccassionalUser** [reply]
Sorry for the late reply.. I have upgraded to the latest version today, had have a plethora of different issues now. I have raised a discussion point on that (latest upgrade, shows same stream from different cameras).  I have had to disable one of the cameras as the zones and masks are all different (as it picks up the stream from the second camera). I shall paste the JSON file as soon as I can sort out the latest issue

**Comment by HAOccassionalUser** [comment]
Hi  Blake,
Trying to get an understanding of when a clip is written to the database. The issue described above (the neighbours parked car is now recognised as a movement ) is causing snapshots to be written almost once every minute or in some cases every 30 seconds. Mostly at night, and I think the issue is that on very windy nights, the security light comes on, and the car is identified. I have disabled alerts as they are no longer practical, but just wondering if I can possibly set something that could reduce the number of snapshots.  There are more than 200 or 300 snapshots by the next morning. 
The only way I could figure out on how to reduce the snapshots is to increase the threshold which is now .95. But that is also not really practical.  
Any other suggestions?
regards
Tom

**Reply by blakeblackshear** [reply]
Try setting your motion detection to this:

```yaml
motion:
  frame_height: 50
  contour_area: 30
```
